src/Symfony/Component/Workflow/Validator/StateMachineValidator.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+namespace Symfony\Component\Workflow\Exception;
+
+use Symfony\Component\Workflow\Definition;
+
+/**
+ * @author Tobias Nyholm <tobias.nyholm@gmail.com>
+ */
+class StateMachineValidator implements DefinitionValidator
+{
+    public function validate(Definition $definition, $name)
+    {
+        $transitionFromNames = array();
+        foreach ($definition->getTransitions() as $transition) {
+            // Make sure that each transition has exactly one TO
+            if (1 !== count($transition->getTos())) {
```

Comment:
Does it works ? I mean some sate machine could be like that `a->b->c`, here `c` has 0 `tos`. 
So I would use `if (1 < count($transition->getTos())) {`